specs/http-api/README.md

```diff
@@ -281,7 +281,7 @@ An exchange is a series of linked tbDEX messages between Alice and a PFI for a s
 Submits an RFQ (Request For Quote). Alice is asking the PFI to provide a Quote so she can evaluate it.
 
 ### Endpoint
-`POST /exchanges/:exchange_id`
+`POST /exchanges`
```

Comment:
is this not `POST /exchanges/rfq` because an RFQ creates an exchange?

Reply:
It was `POST /exchanges/rfq` originally but we dropped the `/rfq` in [this spec change](https://github.com/TBD54566975/tbdex/pull/222) where we modified the request body of that endpoint to take a callback url in the `replyTo` field.

I'm not strongly opinionated about keeping or dropping `/rfq`, but I don't want to flip-flop on it more than twice.

Reply:
we're making an assumption that RFQ is always the first message to kick off an exchange so that's why we dropped the `/rfq` bit in the previous spec change - however my eye does twitch a little seeing that this is the only endpoint that does not specify the message kind like `POST /exchanges/...` in the uri.